client/app/components/AddressForm.jsx

```diff
@@ -7,20 +7,23 @@ import TextField from 'app/components/TextField';
 import SearchableDropdown from 'app/components/SearchableDropdown';
 import { createFilter } from 'react-select';
 
-export const AddressForm = ({ control, register, watch, isOrgPartyType }) => {
+export const AddressForm = ({ control, register, watch, isOrgPartyType, isIndividualPartyType, isHLROrSCForm }) => {
   const watchState = watch('state');
   const defaultState = useMemo(
     () => STATES.find((state) => state.label === watchState),
     [STATES, watchState]
   );
 
+  const optionalIndividualHLRField = Boolean(isIndividualPartyType && isHLROrSCForm);
```

Comment:
```suggestion
  const optionalIndividualClaimReviewField = Boolean(isIndividualPartyType && isHLROrSCForm);
```

I'm wondering if this would be better renamed to reflect its potential use in both claim review intake flows.

Reply:
Thanks for catching this one. I thought I had renamed all the variables. I'll go with HLROrSCField to stick with the already ugly variable naming convention.